overlord/devicestate/firstboot.go

```diff
@@ -70,9 +68,9 @@ func installSeedSnap(st *state.State, sn *snap.SeedSnap, flags snapstate.Flags)
 	return snapstate.InstallPath(st, &sideInfo, path, sn.Channel, flags)
 }
 
-func trivialSeeding(st *state.State, markSeeded *state.Task) []*state.TaskSet {
+func trivialSeeding(st *state.State, baseSnap string, markSeeded *state.Task) []*state.TaskSet {
 	// give the internal core config a chance to run
-	configTs := snapstate.ConfigureSnap(st, "core", 0)
+	configTs := snapstate.ConfigureSnap(st, baseSnap, 0)
```

Comment:
this change doesn't make sense to me, we really want to make the system config either special or attached to snapd, not the base snap. I understand it does the right thing when baseSnap == "core" but it doesn't look like the expecte behavior if base != "core"

Reply:
I agree with this, we said that configuration should live on snapd snap, not on the core/system snap